src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/scripts/fixGencodeOrdering.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python
```

Comment:
Shouldn't this go in the scripts/ directory that starts at the root level?  This is mixed in with the Java code.

Reply:
Yeah - I didn't know there was a scripts directory.  I'll make a funcotator folder there and move it in.